### PR TITLE
Removed /fs/ess/scratch & /fs/project from Files Shortcuts

### PR DESCRIPTION
**Pull request by gerald-byrket**

### CODE REVIEW

File: ondemand.osc.edu/apps/dashboard/initializers/ood.rb [hunk 1]
```diff
@@ -3,14 +3,18 @@ def add_paths
   OodFilesApp.candidate_favorite_paths.tap do |paths|
     # add project space directories
     projects = User.new.groups.map(&:name).grep(/^P./)
-    paths.concat projects.map { |p| Pathname.new("/fs/project/#{p}")  }
+    
+    # Removing /fs/project from list per asana https://app.asana.com/0/1135148780858012/1203345685227840/f
+    # paths.concat projects.map { |p| Pathname.new("/fs/project/#{p}")  }
```
[the remark this answers is not in the document]
Comment (gerald-byrket) [reply ondemand.osc.edu/apps/dashboard/initializers/ood.rb:8]: > Do we need the comments? Can't we just remove?

sure.

File: ondemand.osc.edu/apps/dashboard/initializers/ood.rb [hunk 2]
```diff
@@ -3,14 +3,11 @@ def add_paths
...
-    paths.concat projects.map { |p| Pathname.new("/fs/project/#{p}")  }
-
+    
     # add scratch space directories
     paths << Pathname.new("/fs/scratch/#{User.new.name}")
     paths.concat projects.map { |p| Pathname.new("/fs/scratch/#{p}")  }
 
-    # add ess scratch and project directories
-    paths.concat projects.map { |p| Pathname.new("/fs/ess/scratch/#{p}")  }
     paths.concat projects.map { |p| Pathname.new("/fs/ess/#{p}")  }
```
Comment (johrstrom) [inline ondemand.osc.edu/apps/dashboard/initializers/ood.rb:11]: Are you sure these are the right directories? We're removing `/fs/ess/scratch` but keeping `/fs/ess/<project>`?

Seems like we're being told to get rid of all ess and keep regular fs? But this removes one of each.

Comment (gerald-byrket) [reply ondemand.osc.edu/apps/dashboard/initializers/ood.rb:11]: from Summer.

I will suggest to remove all /fs/project/<project-code> and /fs/ess/scratch/<project-code>

Comment (johrstrom) [reply ondemand.osc.edu/apps/dashboard/initializers/ood.rb:11]: Yea I'm seeing that's how they want it. Fine by me I guess.

**Review by johrstrom — APPROVED** [review]
Works for me if that's what they want.

**Comment by gerald-byrket** [comment]
> Works for me if that's what they want.

Yep. that's what they said they wanted.

`Encourage users to use /fs/ess and /fs/scratch`

**Pull request merged**